KidsIdKit/scss/bootstrap/scss/bootstrap-grid.css

```diff
@@ -1,6 +1,6 @@
 /*!
```

Comment:
Are these all generated .css files? If so they are build artifacts and shouldn't be pushed into source control.

Probably need to exclude the folder into which these are built.

I'd expect there to be soure scss files that we manually maintain, and perhaps css files in the httproot folder that we maintain - anything that is generated by a tool should not go into source code.

Reply:
Oh, @Titangizmo, perhaps you updated us to a newer version of bootstrap and these _are_ source files?

I'm trying to remember exactly how all the css stuff works!